### DISCUSSION TITLE
无效：未阅读说明

### DISCUSSION THREAD

**Discussion by AceCandy** [hooke007/mpv_PlayKit#230] [回收]
使用rife_cuda补帧点击mkv格式电影播放的时候会先弹出一个命令窗口，如下图
![image](https://user-images.githubusercontent.com/6278583/202860503-36076b60-be19-42f7-b75f-2e2820fb832d.png)

而且必须关闭这个窗口，mpv才会正常跳出来播放，是我的设置有问题还是这个rife_cuda就是这样的呢？0.0

**Comment by hooke007** [comment]
请仔细阅读说明文档。关掉这个窗口和不开补帧没有区别
> 需要对不同的分辨率首次运行前，都要预生成一次引擎文件，此步骤视分辨率、模型、显卡性能而定，每次耗时3~10分钟不等。
